src/core_plugins/region_map/public/choropleth_layer.js

```diff
@@ -61,6 +61,9 @@ export default class ChoroplethLayer extends KibanaMapLayer {
 
     this._showAllShapes = showAllShapes;
     this._geojsonUrl = geojsonUrl;
+    this._notifier = notifier;
```

Comment:
I think we could already use the new [toast notifications](https://github.com/elastic/kibana/blob/master/src/ui/public/notify/toasts/TOAST_NOTIFICATIONS.md) for that?

Reply:
okay, change thx